beacon_node/beacon_processor/src/lib.rs

```diff
@@ -615,47 +588,94 @@ pub enum Work<E: EthSpec> {
 
 impl<E: EthSpec> fmt::Debug for Work<E> {
     fn fmt(&self, f: &mut fmt::Formatter<'_>) -> fmt::Result {
-        write!(f, "{}", self.str_id())
+        write!(f, "{}", Into::<&'static str>::into(self.to_type()))
     }
 }
 
+#[derive(IntoStaticStr, PartialEq, Eq, Debug)]
```

Comment:
You can use `#[strum(serialize_all = "snake_case")]` to keep the snake case identifiers

https://docs.rs/strum/latest/strum/additional_attributes/index.html